db/schema.rb

```diff
@@ -23,6 +23,13 @@
     t.string   "gender",                      :null => false
     t.datetime "created_at",                  :null => false
     t.datetime "updated_at",                  :null => false
+    t.integer  "title_id",                    :null => false
+  end
+
+  create_table "titles", :force => true do |t|
+    t.string   "name",       :null => false
+    t.datetime "created_at", :null => false
+    t.datetime "updated_at", :null => false
```

Comment:
Plus development prefers hash syntax over rockets e.g. `null: false` over `:null => false`.
[Rubocop](https://github.com/rubocop/rubocop) will point this out for you by default.

Reply:
Thanks, I keep forgetting.

Reply:
Yeah good point, my bad.